src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -2093,6 +2093,11 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
         }
         const result = await this.query(`SELECT "udt_schema", "udt_name" ` +
             `FROM "information_schema"."columns" WHERE "table_schema" = '${schema}' AND "table_name" = '${name}' AND "column_name"='${column.name}'`);
+        
```

Comment:
Strange that it would end up being `undefined`, the query would surely be identical between the two columns if name is the same?

Reply:
same comment here

Reply:
Have you tried to reproduce it?

That's the behavior I encountered.

Imo you should think about using typescripts `--noUncheckedIndexedAccess`.